tables/atom.py

```diff
@@ -1078,12 +1192,9 @@ class VLStringAtom(_BufferedAtom):
     type = 'vlstring'
     base = UInt8Atom()
 
-    def _tobuffer(self, object_):
-        if isinstance(object_, str):
-            warnings.warn("Storing non bytestrings in VLStringAtom is "
-                          "deprecated.", DeprecationWarning)
```

Comment:
We should have an entry in the release notes for this change.

Reply:
Added a small section in the release notes.